src/ECU.cpp

```diff
@@ -3,20 +3,17 @@
 #include "Throttle.h"
 #include "Brake.h"
 
```

Comment:
I'm curious about whether we should include these min/max IDs into the enum and CAN msg datasheet?

Reply:
we do I forgot to delete